File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L3/01/L3.ts
import * as readline from 'readline';

function balik(nomor: number): void {
    let sisa: number = 0;
    while (nomor !== 0) {
        sisa = nomor % 10;
        console.log(sisa);
        nomor = Math.floor(nomor / 10);
    }
    console.log();
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter an integer: ', (input) => {
    const nomor: number = parseInt(input);
    balik(nomor);
    rl.close();
});

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L3/01/L3.kt
package IR

fun main() {
    var nomor = 0
    println("Enter an integer: ")
    
    val input = readln().toInt()
    
    balik(input)
}

fun balik(nomor: Int) {
    var sisa = 0
    while (nomor != 0) {
        sisa = nomor % 10
        print(sisa)
        nomor /= 10
    }
    println()
}

